dashboard/templates/dashboard/clustal_hist.html

```diff
@@ -86,7 +88,7 @@ <h4 class="panel-title">
                         <div>
                             <a href="{% url 'dashboard_blast' %}" class="list-group-item">BLAST</a>
                             <a href="{% url 'dashboard_hmmer' %}" class="list-group-item">HMMER</a>
-                            <a href="{% url 'dashboard_clustal' %}" class="list-group-item">CLUSTAL</a>
+                            <a href="{% url 'dashboard_clustal' %} " class="list-group-item">CLUSTAL</a>
```

Comment:
I think this line is accidentally edited, should be changed back to original one.